build.rb

```diff
@@ -24,6 +23,8 @@
   ruby
   nvm
   yarn
+  macos_sqlite
+  vscode_extensions
```

Comment:
I'ld rather keep the `vscode_extensions` partial just after the VS Code installation instructions partial, where it was. Do we need to have SQLite already installed when we install the VS Code SQLit extension? :thinking:

Reply:
I've uninstalled sqlite and the extension on my Linux machine and I can install back the extension without having sqlite binay installed yet. It's not working of course but the extension installation process is still working. Let's restore the `vscode_extensions` position then :pray: